lib/flow.js

```diff
@@ -99,6 +99,8 @@ const glob = function glob(cb) {
   }.bind(this));
 };
 
+const removeFileExtension = filename => filename.endsWith('.js') ? filename.substr(0, filename.length - 3) : filename;
```

Comment:
There appears to be little benefit in pulling up this one line of code into its own function, unless it may potentially be used elsewhere. It's fine to wait until such time as that is the case before doing this.